test/distributed/org/apache/cassandra/distributed/upgrade/MixedModeConsistencyTest.java

```diff
@@ -38,9 +38,10 @@
 import static org.apache.cassandra.distributed.shared.AssertUtils.assertRows;
 import static org.apache.cassandra.distributed.shared.AssertUtils.row;
 
-public class MixedModeConsistencyTestBase extends UpgradeTestBase
+public class MixedModeConsistencyTest extends UpgradeTestBase
 {
-    protected static void testConsistency(Semver initial) throws Throwable
+    @Test
+    public void testConsistency() throws Throwable
```

Comment:
there used to be a single subclass for `v30`. I removed it and just inlined `v30`.
I don't see value in a separate class (unless we care for butler history)